#Request

bool request

---

#Response

geometry_msgs/Pose ee_position
